enhancement/9post-map

Dongha922 opened the pull request:
map 생성

Dongha922 committed:
api 발급

Dongha922 committed:
api 발급

Dongha922 committed:
api 발급

Dongha922 committed:
api 발급

Dongha922 committed:
지도생성

Dongha922 committed:
지도생성

Dongha922 committed:
지도생성

Dongha922 committed:
지도생성

Dongha922 committed:
지도생성

Dongha922 committed:
지도생성

Dongha922 committed:
지도 키워드검색목록완성

Dongha922 committed:
지도 키워드검색목록완성

Dongha922 committed:
지도 키워드검색목록완성

Dongha922 committed:
지도 키워드검색목록완성

Dongha922 committed:
ãhomecontrolleãr 충돌

Dongha922 committed:
ãhomecontrolleãr 충돌

Dongha922 committed:
ãhomecontrolleãr 충돌

Dongha922 committed:
ãhomecontrolleãr 충돌

Dongha922 committed:
Merge branch 'e/9' of https://github.com/Tour-de-Monde/Tour-de-Monde into e/9

Dongha922 committed:
ããmap충돌해결

Dongha922 committed:
ããmap충돌해결

Dongha922 committed:
Merge branch 'e/9' of https://github.com/Tour-de-Monde/Tour-de-Monde into e/9

AidennnLee committed:
refactor : 코드 수정

AidennnLee committed:
comment : 주석 추가

Son-Gyeongi commented on src/main/java/com/ll/tourdemonde/HomeController.java:
```diff
@@ -0,0 +1,24 @@
+import org.springframework.web.bind.annotation.ResponseBody;
+@Controller
+
+public class HomeController {
```
동하님 HomeController가 다른게 존재해서 해당 컨트롤러는 지울 수 있을까요?

Dongha922 replied:
삭제완료

Son-Gyeongi commented on src/main/resources/application-test.yml:
```diff
@@ -1,3 +1,26 @@
+#spring:
```
동하님 application-test.yml에 중복되는 부분은 지워야 할 거 같습니다.

Dongha922 replied:
주석삭제 완료

Son-Gyeongi commented on src/main/resources/application.yml:
```diff
@@ -1,3 +1,48 @@
+#spring:
```
application.yml도 중복되는 부분은 지워야 할 거 같습니다.

Dongha922 replied:
주석삭제 완료

Son-Gyeongi reviewed (commented):
다 잘하셨습니다! 수정해야 할 부분만 수정하면 될 거 같습니다.

Dongha922 committed:
주석처리, 홈컨트롤러삭제

Dongha922 committed:
주석처리, 홈컨트롤러삭제

geniushee committed:
Enhancement/6 - 예약 CRUD 기능 구현 (#13)

* save : Place테이블 생성

* feat : 장소 Create 기능 구현 완료
- 장소 저장 잘됨
- 같은 장소 두번 저장되는 현상 해결해야 함

* comment : 코드 정리

* feat : 장소 여러개 저장

* Build : 의존성 추가 및 yml파일 업데이트

* refactor : 장소가 이미 존재 하는 경우, 장소를 저장 하지 않음
- 장소 1개 등록 엔드포인트 필요없어서 제거
- application.yml 수정, application-test.yml 추가

* refactor : 장소에 좌표(coordinates)를 String 타입으로 변경

* feat : reservation 및 reservationDetail Entity 추가

* conflict : main을 enhancement/1에 rebase했을 때 충돌 해결

* conflict : main을 enhancement/1에 rebase했을 때 충돌 해결

* comment : 코드 정리

* feat : 장소 여러개 저장

* refactor : 장소가 이미 존재 하는 경우, 장소를 저장 하지 않음
- 장소 1개 등록 엔드포인트 필요없어서 제거
- application.yml 수정, application-test.yml 추가

* refactor : 장소에 좌표(coordinates)를 String 타입으로 변경

* test : test를 위한 place 데이터 추가

* conflict : enhancement/1에 main을 rebase 후 충돌 해결 2

* conflict : enhancement/1에 main을 rebase 후 충돌 해결 4

* conflict : enhancement/1에 main을 rebase 후 충돌 해결 5

* refactor : 장소에 좌표(coordinates)를 String 타입으로 변경

* feat : 장소 이름과 장소 주소로 장소 찾기

* feat : 장소 삭제 추가
- '로그인한 회원만 글쓰기 접근 가능'은 아직 구현이 안 했음
- 프론트엔드는 아직 구현 안함

* feat : 예약페이지 UI초안 추가

* feat : reservationCreateForm 및 reservationType 추가

* feat : 장소 수정 추가

* feat : reservation 저장 기능 및 관리페이지 생성

* feat : createReservation form 생성 및 save 기능 완료

* test : 장소 조회 GET /place

* feat : createReservationDetail 페이지 form 추가 및 postmapping 추가

* test : 장소 등록 POST /place/save 중간 저장

* test : 장소 삭제 POST /place/delete

* feat : reservation option 추가 및 페이지 구현

* test : 장소 수정 POST /place/modify

* test : 장소 삭제 테스트 검증 추가

* comment : 코드 정리

* feat : 예약 조회를 위한 예약페이지 개선

* feat : reservation별로 옵션 조회기능 추가

* feat : 장소 찾기 페이지 구현

* refactor : place 찾기 기능 복원

* fix : test.yml 수정

* faet : 예약 관리페이지 생성

* feat : modify reservation 추가

* refactor : 수정기능 return값 변경 및 일부 endPoint 가독성 향상

* fix : 새 예약 생성 간단한 버그 수정

* feat : 옵션 수정 기능 구현, 에러발생

* refactor : 옵션 수정 기능 구현 완료 및 refactoring

* feat : 예약 삭제 구현

* feat : 옵션 삭제 기능 구현 완료

* feat : order entity 생성

* test : 테스트 추가, 에러 발생

* fix : JBDC error 해결

* test : GET '/reserve/1' 테스트 실시 및 테스트중 에러발생 수정

* test : 예약 생성 post 테스트 생성

* test : 예약 옵션 생성 GET, POST 테스트 생성

* test : 예약 수정 GET 테스트 생성

* test : 예약 수정 POST 테스트 생성

* test : 예약 CRUD 기능 테스트 완료

* refactor : pakage 일부 변경

* fix : 원격 main 리포지토리 pull & merge

---------

Co-authored-by: Son-Gyeongi <thsruddl77@gmail.com>

AidennnLee committed:
refactor : 코드 수정

geniushee committed:
Enhancement/6 - 테스트로 인해 발생하는 에러 처리 완료 (#14)

* hotfix : initdata 미성으로 인한 테스트 안되던 문제 해결

---------

Co-authored-by: Son-Gyeongi <thsruddl77@gmail.com>

AidennnLee committed:
채comment : 주석 추가

AidennnLee committed:
recomment : 주석 추가

AidennnLee committed:
"refactor : 코드 수정"

Dongha922 committed:
search를 post_create로 이동

Dongha922 committed:
search를 post_create로 이동

geniushee reviewed (approved):
1. 이후에 search.html에 있는 style은 별도의 css 파일로 분리하는것도 괜찮을 것 같습니다.(강요는 아닙니다;;)
2. 새로운 API 공부하고 적용하시는데 고생하셨습니다.

AidennnLee committed:
Merge pull request #15 from Tour-de-Monde/enhancement/5

Enhancement/5

Dongha922 committed:
relayout(), setTimeout으로 ãmapResize추가

Son-Gyeongi committed:
refactor : Dockerfile 소스 코드 복사 (테스트 폴더 제외) 수정

Dongha922 committed:
relayout(), setTimeout으로 ãmapResize추가

Dongha922 committed:
불필요 컨트롤ㄹ러삭제

Dongha922 committed:
컨틀롤러 나중엣 ã ãdelete

Dongha922 committed:
ãmap 자리 이동

Dongha922 committed:
map수정

Dongha922 committed:
지도 밀기
